lib/ansible/modules/system/aix_nimclient.py

```diff
@@ -0,0 +1,545 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# Copyright: (c) 2017, Joris Weijters <joris.weijters@gmail.com>
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+ANSIBLE_METADATA = {'metadata_version': '1.1',
+                    'status': ['preview'],
+                    'supported_by': 'community'}
+
+DOCUMENTATION = '''
+---
+author: "Joris Weijters (@molekuul)"
+module: aix_nimclient
+short_description: installs software using the nimclient, and removes software
+description:
+    - installs software at the nimclient using nimclient command and removes software using local commands also updates ALL to latest version
+version_added: "2.8"
+options:
+  name:
+    description:
```

Comment:
```suggestion
    description:
    - Name of the fileset to install.
    - Use C(update_all) to update all filesets to latest level.
    type: list
```
Other modules use `*` or `all`, so maybe that's a better convention to use ?